generators/app/templates/graphQL/.woloxci/config.yml

```diff
@@ -0,0 +1,28 @@
+config:
```

Comment:
do we need another template for woloxci?

Reply:
ok!